Src/Support/Google.Apis.Auth/ExistingDependencies/ReplacementSerializer.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Google.Apis.Auth.OAuth2;
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Google.Apis.Auth.ExistingDependencies;
+
+// The replacement for NewtonsoftJsonSerializer
+internal class ReplacementSerializer
```

Comment:
Is the idea to take a dependency on System.Text.Json or to just serialize/deserialize by hand?

Reply:
I have no opinion on that at the moment :)